lib/ecosystems/enrich_spdx_test.go

```diff
@@ -61,7 +61,7 @@ func TestEnrichSBOM_SPDX(t *testing.T) {
 	}
 	doc := &sbom.SBOMDocument{BOM: bom}
 
-	EnrichSBOM(doc)
+	EnrichSBOM(doc, &logger)
```

Comment:
In this file the `logger` is not actually defined anywhere, also somehow this didn't make the test crash

Reply:
defined in the cyclonedx test file, they have the same package name. do you think we should move it to a different (more neutral) file?

Reply:
actually, we don't use the _test naming convention here, this results in ugly shadowing. will create a logger in every test function.

Reply:
oh, makes sense now